workspace_integration_test.go

```diff
@@ -414,6 +413,18 @@ func TestWorkspacesCreate(t *testing.T) {
 		assert.EqualError(t, err, ErrUnsupportedBothTagsRegexAndFileTriggersEnabled.Error())
 	})
 
+	t.Run("when options include both non-empty tags-regex and file-triggers-enabled as false an error is not returned", func(t *testing.T) {
```

Comment:
suggestion: We could use table driven tests and merge together this and the above 3 tests

Reply:
@hashimoon We're good with adding table driven tests here - agree that it would help readability.

Reply:
Given the rest of the file is not utilizing table driven tests, can we add them in a future refactor? @annawinkler

Reply:
Given the rest of the file is not utilizing table driven tests and the style is consistent with what is already present, can we add them in a future refactor? @annawinkler